### DISCUSSION TITLE
Standup (Monday, April 25, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#20] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - I began drawing the activity diagram #15 😎.

**What are your priorities for the next 24 hours?**:
 - Conclude the diagram #15 

**Blockers**:
 - Poor understanding of requirements

**Shoutouts**:
 - @DavBot02 , @CarlosCRG19, @CristianCazares  and @paotg123 for being so communicative <3

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing really

**What are your priorities for the next 24 hours?**:
 - Continue with #14 
 - Meet with team in order to define requirements

**Blockers**:
 - The final requirements for this iteration must be define before we continue with use cases and activity diagrams, IMO.

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing 😯

**What are your priorities for the next 24 hours?**:
 - Help @antonieto with #15 

**Blockers**:
 - Define requirements

**Shoutouts**:
 - @everyone for being so hard-working

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - zzz

**What are your priorities for the next 24 hours?**:
 - Conclude the introduction??? Finally??? (#12 )

**Blockers**:
 - Redefine the "problem" or something?
